Update fund.mdx with Sepolia details (#505)
Consensys/doc.zk-evm: docs/use-mainnet/fund.mdx

- [Faucets](#use-a-linea-faucet) (for testnet tokens on Linea Goerli)
- [Joining and asking in the Linea Builders Club](https://linea.deform.cc/linea-builders-club)
- [Bridging between Goerli and Linea Goerli](./bridges-of-linea/index.mdx)
- Sending a DM to [this XMTP chatbot](https://converse.xyz/dm/0x96561c772533d32da71d240820a2fbf6883cf96a) 
at TheGeneralStore.eth

## Use a Linea faucet

> If you want to drip Goerli ETH directly to Linea testnet, the following faucets are available. Note
> that you will need to enter your actual address — ENS names will not work.

Reviewer: ```suggestion
If you want to drip Goerli ETH directly to the Linea testnet, use the following faucets. Note that you must enter your actual address — ENS names will not work.
```

1. [Infura Linea faucet](https://infura.io/faucet/linea)
2. [Covalent Linea faucet](https://www.covalenthq.com/faucet/)
3. [FAUCETME faucet](https://linea.faucetme.pro/)
4. [LearnWeb3 Faucet](https://learnweb3.io/faucets/linea_goerli)

> If you want more ETH than the daily allotted amount or run into trouble with the above faucets, you
> can also [bridge ETH to Linea](/use-mainnet/bridges-of-linea/how-to-bridge-eth). Note that you'll 
> need to [get test ETH on Goerli](#get-test-eth-on-goerli) in order to do so.

Reviewer: ```suggestion
If you want more ETH than the daily allotted amount, or run into trouble with the above faucets, you
can also [bridge ETH to Linea](/use-mainnet/bridges-of-linea/how-to-bridge-eth). Note that you'll 
need to [get test ETH on Goerli](#get-test-eth-on-goerli) to do so.
```

> If you want to drip other tokens, you can find the [multi-token Linea faucet here](https://faucet.goerli.linea.build/), 
> which lists the different tokens you can add to your wallet on the Goerli and Linea Goerli testnet.

Reviewer: ```suggestion
If you want to drip other tokens, use the [multi-token Linea faucet](https://faucet.goerli.linea.build/), 
which lists the different tokens you can add to your wallet on the Goerli and Linea Goerli testnet.
```

   your bank's location.
4. **Set the payment method**. Again, if you've used Buy before on your current device, your 
previous choice should be loaded by default.
5. **Select a token**. The token selection menu also allows you to switch networks; it's here 
where you need to locate and select Linea from the networks list.
6. Now you'll see the tokens available to buy on Linea. Pick the one you want to buy, then input 
the amount on the main screen once the tokens menu is gone.
7. You'll instantly see one or more quotes on the right, if you're on a desktop browser. On mobile 
devices, hit 'Get Quotes' to see them.
8. Pick a quote by clicking on it.

Selecting a quote redirects you to the provider's site to complete the transaction. You may need 
to complete KYC ("Know Your Customer"; essentially, identifying yourself) with the provider.

After you've completed the steps with the provider, you'll be routed back to MetaMask Portfolio. 
Timescales for receiving your funds will vary based on the provider and token route.

</TabItem>

<TabItem value="Linea Sepolia">

## Linea Sepolia faucets

> An up-to-date list of available Linea Sepolia faucets is available [here](../build-on-linea/goerli-to-sepolia.mdx#faucets).

Reviewer 2: Is there a reason we are redirecting to the other page rather than putting it directly on this page

Author: No reason other than it'll prevent us having to update the information in two places. The faucets aren't all currently live so the information will need updating in `goerli-to-sepolia` at some point. Appreciate though that being sent round the docs isn't ideal UX.
